extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -183,8 +183,9 @@ export class AmpSidebar extends AMP.BaseElement {
     }
 
     if (!element.hasAttribute('role')) {
-      element.setAttribute('role', 'menu');
+      element.setAttribute('role', 'dialog');
```

Comment:
This is potentially a breaking change if people use `role=menu` to set styling. Do we know why we're making this change? Or, is this a change that ideally only applies when `amp-nested-menu` is a child component of `amp-sidebar`?

Reply:
This is coming from https://github.com/ampproject/amphtml/issues/24382 and that's true, it's a breaking change. Not sure how to proceed here.

Reply:
To be on the safe side, I would recommend gating this change under an aggressively rolled out experiment (you can directly set it to 100% and monitor for two release cycles), or putting it in an isolated PR that allows a clean rollback, but feel free to use your discretion.